src/pint/pintk/plk.py

```diff
@@ -2,32 +2,28 @@
 Interactive emulator of tempo2 plk
 """
 import copy
+import logging
 import os
 import sys
 
 import astropy.units as u
 import matplotlib as mpl
 import numpy as np
-from astropy import log
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
 
 import pint.pintk.pulsar as pulsar
 import pint.pintk.colormodes as cm
 
-try:
-    # Python2
-    import Tkinter as tk
-    import tkFileDialog
-    import tkMessageBox
-except ImportError:
-    # Python3
-    import tkinter as tk
-    import tkinter.filedialog as tkFileDialog
-    import tkinter.messagebox as tkMessageBox
-    from tkinter import ttk
+import tkinter as tk
+import tkinter.filedialog as tkFileDialog
+import tkinter.messagebox as tkMessageBox
+from tkinter import ttk
+
+log = logging.getLogger(__name__)
 
-log.setLevel("INFO")
-log.debug("This should show up")
+# FIXME: do not set logging in an importable module. Why is this here? settings should go in the scripts/
```

Comment:
The setLevel line was added by @aarchiba  in 2019, and the log.debug line was added by @champagne-cmd . I agree that they can probably both be removed.